api-platform/core/2.1/post-install.txt

```diff
@@ -0,0 +1,12 @@
+<bg=blue;fg=white>              </>
+<bg=blue;fg=white> What's next? </>
+<bg=blue;fg=white>              </>
+
+  * Your API is almost ready:
+    1. Create your first API resource in <info>src/Entity</info>;
```

Comment:
What do you thing about adding an example using MakerBundle here? (ping @weaverryan)

Reply:
Ah, it’s tough because we don’t know if it’s installed. And, maybe the user already has an entity that they want to turn into a Resource? You would know better than me what a typical use-case is :)

Also, a new user won’t really know what to do at this first step - they won’t know how to “create an API” resource. A link to some very direct docs might be better?

Reply:
So maybe:

    Create a class in <info>src/Entity</info> and mark it with the <info>ApiPlatform\Core\Annotation\ApiResource</info> annotation.
    Alternatively, if <info>symfony/maker-bundle</info> is installed, run <info>bin/console make:entity --api-resource</info> to generate an API resource class interactively.

Reply:
It’s really long, and you’ll normally add a use statement for the annotation anyways.

I think it’s better to describe what to do “mark an entity as an ApiResource. See https://“.